mathics/builtin/atomic/numbers.py

```diff
@@ -948,6 +934,23 @@ class Precision(Builtin):
     >> Precision[{{1, 1.`},{1.`5, 1.`10}}]
      = 5.
 
+    For non-zero Real values, it holds in general
+
+    $$Accuracy[z] == Precision[z] + Log[z]$$
```

Comment:
This does not render right in Django. I don't think we handle `$$`.  It would be the more awkward: 

```
'Accuracy'[$z$] == Precision[$z$] + 'Log[$z$]'
```

Sigh. When we move to a sphinx or a real documentation system, it is going to be a major pain to go back and simplify all of our home-grown tagging.

Reply:
There are a number of changes that are marked "resolved" that I am not seeing. This is one.

Reply:
Sorry, I tried to go one by one, correct, and push here before marking them. I am going over it again.